src/mesos.ts

```diff
@@ -186,15 +187,22 @@ function taskToTaskInfo(task: MesosTask, state: MesosState): TaskInfo {
   const runningStatuses = task.statuses.filter(s => s.state === 'TASK_RUNNING');
```

Comment:
Maybe that's a naive comment, but maybe the fact that we can't browse failed sandbox is due to the fact that we filter with status TASK_RUNNING?

A clearer approach would be, imho, to fallback to non running state to browser sandboxes and block access to the terminal in that case.

Reply:
I thought of that but actually the non running status in meoss master state does not container the container.